DigitalLearningSolutions.Web/Views/LearningMenu/Diagnostic/Diagnostic.cshtml

```diff
@@ -0,0 +1,86 @@
+  ViewData["Title"] = "CMS Demonstration Course - Vimetube test";
+  ViewData["CustomisationId"] = 1;
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningMenu/{1}";
+  ViewData["HeaderPathName"] = "CMS Demonstration Course - Vimetube test";
+}
+@section NavMenuItems {
+  <partial name="Shared/_NavMenuItems" />
+}
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading" class="nhsuk-heading-xl">Working with Microsoft Office applications</h1>
+    <h2 class="nhsuk-u-secondary-text-color">Not attempted</h2>
+  </div>
+</div>
+
+<div class="nhsuk-inset-text nhsuk-u-margin-top-0 nhsuk-u-margin-bottom-4">
+  <h3 class="nhsuk-heading-s">Important information:</h3>
+  <ul class="nhsuk-list nhsuk-list--bullet">
+    <li>Your score will be reset for any areas that you have attempted previously and choose to attempt again.</li>
+    <li>Please allow sufficient time to complete the assessment. If you close part way through, your progress will not be stored.</li>
+    <li>Throughout most assessments, green or red indicators will appear to indicate whether you have answered correctly.</li>
+    <li>Most keyboard shortcuts will be disabled in the assessment because we are unable to capture them in the testing software. If in doubt, use the mouse.</li>
+  </ul>
+</div>
+
+<form class="nhsuk-form-group">
+  <fieldset class="nhsuk-fieldset">
+    <legend class="nhsuk-fieldset__legend nhsuk-fieldset__legend--l">
+      Which areas do you want to assess?
+    </legend>
+
+    <div class="nhsuk-hint" id="contact-hint">
+      Select all areas that you want to test.
+    </div>
+
+    <div class="nhsuk-checkboxes__item">
+      <input class="nhsuk-checkboxes__input" type="checkbox" checked="checked" id="1" value="1">
+      <label class="nhsuk-label nhsuk-checkboxes__label" for="1">
+        Introduction to applications
+      </label>
+    </div>
+
+    <div class="nhsuk-checkboxes__item">
+      <input class="nhsuk-checkboxes__input" type="checkbox" checked="checked" id="2" value="2">
+      <label class="nhsuk-label nhsuk-checkboxes__label" for="2">
+        Common screen elements
+      </label>
+    </div>
+
+    <div class="nhsuk-checkboxes__item">
+      <input class="nhsuk-checkboxes__input" type="checkbox" checked="checked" id="3" value="3">
+      <label class="nhsuk-label nhsuk-checkboxes__label" for="3">
+        Using ribbon tabs
+      </label>
+    </div>
+
+    <div class="nhsuk-checkboxes__item">
+      <input class="nhsuk-checkboxes__input" type="checkbox" checked="checked" id="4" value="4">
+      <label class="nhsuk-label nhsuk-checkboxes__label" for="4">
+        Getting help
+      </label>
+    </div>
+
+    <button class="nhsuk-button nhsuk-u-margin-top-3 nhsuk-u-margin-bottom-0" type="submit">
+      Start assessment
+    </button>
+  </fieldset>
+</form>
+
+<div class="nhsuk-back-link">
+  <a class="nhsuk-back-link__link">
```

Comment:
Could set the link destination here, using hardcoded customisation and section ids 🙂